src/sss_client/ssh/sss_ssh_knownhostsproxy.c

```diff
@@ -17,8 +17,33 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
+#include "config.h"
 
 #include <stdio.h>
+#include <stdlib.h>
+
+
+#ifndef BUILD_SSH_KNOWN_HOSTS_PROXY
+
+static const char message[] =
+    "\n"
+    "******************************************************************\n"
+    "You are using the deprecated tool sss_ssh_knownhostsproxy.\n"
```

Comment:
-> "Your system is configured to use `sss_ssh_knownhostsproxy` tool that has been removed. ..."

Reply:
Done